elements/lisk-transactions/src/sign.ts

```diff
@@ -221,15 +111,10 @@ export const signTransactionWithPrivateKey = (
 		throw validationErrors;
 	}
 
-	const transactionWithNetworkIdentifierBytes = Buffer.concat([
-		networkIdentifier,
-		getSigningBytes(transactionObject, paramsSchema),
-	]);
-
 	const signature = ed.signDataWithPrivateKey(
 		TAG_TRANSACTION,
 		networkIdentifier,
-		transactionWithNetworkIdentifierBytes,
```

Comment:
This was a bug because networkIdentifier is concat in the `signDataWithPrivateKey` internally